### PR TITLE
[Docs] Document cli commands for publish rate feature

### PR DESCRIPTION
**Pull request by shiv4289**
cli commands introduced in https://github.com/apache/pulsar/pull/3986 are not documented. Adding documentation for the same.

### CODE REVIEW

**Comment by sijie** [comment]
@shiv4289 can you add the change to 2.5.1 website?

File: site2/docs/reference-pulsar-admin.md [hunk 1]
```diff
@@ -1641,6 +1641,27 @@ Options
...
 
+### `get-publish-rate`
+Get the message publish rate for each topic in a namespace, in bytes as well as messages per second 
+
+Usage
+```bash
+$ pulsar-admin namespaces get-publish-rate tenant/namespace
+```
+
+### `set-publish=rate`
```
Comment (vzhikserg) [inline site2/docs/reference-pulsar-admin.md:1652]: Should not it be `set-publish-rate`? 🤔

File: site2/docs/reference-pulsar-admin.md [hunk 2]
```diff
@@ -1641,6 +1641,27 @@ Options
...
+Usage
+```bash
+$ pulsar-admin namespaces set-publish-rate tenant/namespace options
+```
+
+Options
+|Flag|Description|Default|
+|----|---|---|
+|`-m`, `--msg-publish-rate`|Threshold for number of messages per topic in the namespace (-1 implies not set, 0 for no limit).|-1|
+|`-b`, `--byte-publish-rate`|Threshold for number of messages per topic in the namespace (-1 implies not set, 0 for no limit).|-1|
```
Comment (vzhikserg) [inline site2/docs/reference-pulsar-admin.md:1664]: The comment is copied from the line above and doesn't explain correctly what this parameter does

**Comment by shiv4289** [comment]
> @shiv4289 can you add the change to 2.5.1 website?

Hi @sijie 2.5.0 is where the original feature was added. So, I added docs to all three of:

1. main website
2. 2.5.0 website
3. 2.5.1 website

Please take a look.

**Pull request merged**